packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -189,13 +193,10 @@ export class FASTTreeItem extends FASTElement {
      *
      * @internal
      */
-    public childItemLength(): number {
-        const treeChildren: HTMLElement[] = this.childItems.filter(
-            (item: HTMLElement) => {
-                return isTreeItemElement(item);
-            }
-        );
-        return treeChildren ? treeChildren.length : 0;
+    public get childItemLength(): number | void {
+        if (this.$fastController.isConnected) {
```

Comment:
any specific reason this returns `void` now instead of `0` in a default case? It seems like it might be less error-prone to make this always return a number.

Reply:
good call out...I think this makes sense

Reply:
@nicholasrice...realizing as I went to make the change that the return value would be void if the controller is not connected...not sure we want to return 0 there. Thoughts?

Reply:
Updated to return 0 if not connected, as long as we check for it this seems reasonable. Also will keep typings easy.